about/why-ambassador.md

```diff
@@ -46,14 +46,14 @@ Ambassador takes full advantage of Kubernetes and Envoy Proxy.
 
 ## For more information
 
-[Deploy Ambassador today](https://www.getambassador.io/user-guide/install) and join the community [Slack Channel](http://d6e.co/slack).
+[Deploy Ambassador today](../../user-guide/install) and join the community [Slack Channel](http://d6e.co/slack).
 
-If you're interested in commercial support and additional features, see [Ambassador Pro](https://www.getambassador.io/pro).
+If you're interested in commercial support and additional features, see [Ambassador Pro](/pro).
```

Comment:
should this go to an aes link instead?

Reply:
No, because this is for the current GA version of the docs